### ISSUE TITLE
 ''This project has no readme yet.''

### ISSUE THREAD

**Issue by kolavq** [pepijndevos/spacefox#2]
if the pcb was saved in the link, it does not show the ready project to be ordered.

''This project has no readme yet.''

**Comment by pepijndevos** [comment]
I'm sorry I don't understand what you mean

**Comment by kolavq** [comment]
I apologize for the problem I'm using a translator. When I click on the link from pcb in Bill of Materials, no ready-made board for the project pops up.
![Przechwytywanie](https://user-images.githubusercontent.com/43578980/226116983-95211d5c-3e56-4148-bd0f-0f6df56f489a.PNG)

**Comment by pepijndevos** [comment]
Yes I think you have to make an account with Aisler, and click import.
